src/insns/cbuildcap_32bit.adoc

```diff
@@ -20,19 +24,20 @@ sealed, `cs1` 's permissions and bounds are equal or a superset of `cs2` 's,
 `cs2` 's bounds are not malformed (see
 xref:section_cap_malformed[xrefstyle=short]), and all reserved bits in `cs2` 's
 metadata are 0. <<CBUILDCAP>> is typically used alongside <<CSETHIGH>> to build
-capabilities from integer values.
+capabilities from integer values. The case `cs1 == 0` trivially leads to the tag
```

Comment:
Should this be the canonical ccleartag? If so could add a note suggesting this case is optimized in the uarch

Reply:
I think so. This would add a pseudo instruction `ccleartag cd, cs1` which evaluates to `cbuildcap cd, c0, cs1`

Reply:
I'll resolve this for merge since @arichardson created that as a separate issue.